### DISCUSSION TITLE
Ring Panic Button Doesn't show up in MQTT

### DISCUSSION THREAD

**Discussion by JayLurk** [tsightler/ring-mqtt#856] [General Discussion]
***UPDATE*** After reading through the documentation and FAQ's for the addon, looks like I figured it out. I checked my Addon-Config and Panic was already enabled, however, I had to go to my Alarm device, then check the settings and I saw that the Panic Sliders were disabled. It appears that a Panic Button is not even necessary in this. You can trigger the siren through HA once you enable them. It doesn't appear that the Panic Buttons can be added to HA, which I suppose is essentially irrelevant since you can simply enable them via the Panic Sliders. I only have Self Monitoring, with Ring Protect Plus, so I dont have to worry about Police, Fire/Medical being dispatched to my home.

I've been using the Ring-MQTT Addon for several months now, its fantastic. I wanted the option to be able to trigger the Alarm via Home Assistant. Based on what I gathered, this is not possible with just the Base Station. To make this work I would need a Panic Button. I purchased one of those and a Co/Smoke Listener. Added the Panic button to Ring through the App, its there, but nothing shows up in the MQTT on HA for it. Tried restarting/Rebooting, Stopping Addon, Restarting Addon To No avail, however, when I added the Smoke/Co listener, it was available right away. A

Am I missing something, or is the Panic button not actually supported to use in HA, therefore it wont show up in MQTT?

I currently have Ring Protect Plus plan. The panic switch will set siren off if I press and hold it. Just trying to use it as a way to trigger the alarm for other stuff that isn't native to the Ring system.

I searched the discussion threads and only saw one other topic and it mentioned the Panic switch would be required to actually trigger the siren with HA.

Any help be greatly appreciated, thanks!

**Comment by tsightler** [comment]
Yes, your update is correct, there's no requirement for a panic button, only the panic switches need to be enabled in the config.  They are disabled by default because they are easy to misuse since they are just a switch so automations that do things like turn all switches in a room on will trigger the alarm (quite common with assistants, etc).  Plus, I want people to have to explicitly enable them because it's really easy to create automations that trigger police/fire response and, if you did have monitoring, this is likely a violation of Ring terms of service and, in some jurisdiction which regulate monitored alarm systems due to high rates of false alarm, it might even violate laws that require alarm companies to certify all devices that have the ability to trigger alarm response.

**Reply by tsightler** [reply]
Also, yes, panic buttons are not supported by the addon, mainly because I can't figure out what you would do with one from an integration perspective plus I've never actually seen one or seen a device discovery for one.

**Reply by JayLurk** [reply]
Thanks for the reply. Absolutely killer add on. Good chunk of my home automations revolve around my ring system. Thank you for everything you do!
